x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/exfiltration_rds_snapshot_export.json

```diff
@@ -22,7 +22,7 @@
     "https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_StartExportTask.html"
   ],
   "risk_score": 21,
-  "rule_id": "119c8877-8613-416d-a98a-96b6664ee73a5",
+  "rule_id": "119c8877-8613-416d-a98a-96b6664ee73a",
```

Comment:
What's going on with this rule? New `id`/`version` reset, but everything else is the same? We just abandoning the previous rule for some reason?

Reply:
It was using an improper UUID format. (extra digit). We added tighter validation for this in the repo and deprecated the old rule. Because of similarities, github is just relating it as a change.

Reply:
elastic/detection-rules/pull/1396

Reply:
Yeah, there isn't strict Kibana validation with these ID's as we left it lax to support third party systems/rules:

https://github.com/elastic/kibana/blob/103869509c9e2253499c662a22001e38b06e0088/x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts#L90-L91

That said, with this ID change user's will now have two rules with that name once upgrading. We need a way of deleting/migrating rules in these instances it feels like.

Reply:
++ to adding a way to annotate a rule is deprecated (as well as some UX for removing or keeping it)